src/util/transactions.ts

```diff
@@ -46,6 +47,21 @@ export const parseListTxsRpcTx = tx => {
   ];
 };
 
+export const parseBlockchainTransaction = (tx, address: string) => {
+  const iAmSender = tx.senders.find(s => s === address);
+  return [
+    {
+      value: toBitcoin(tx.value),
+      from: tx.senders,
+      recipient: tx.recepients,
+      time: moment.unix(tx.time).format('DD/MM/YYYY H:mm:ss'),
```

Comment:
1. Let's not use `moment`, `date-fns` is much better.
2. Probably better to just leave this as a unix timestamp and only format directly in components where we want to display it, which allows us to more easily display in different ways in different places.